opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -85,8 +85,8 @@
 try:
     lib_path = os.environ.get("OTIO_AAF_PYTHON_LIB")
     if lib_path and lib_path not in sys.path:
-        sys.path += [lib_path]
-    import aaf # noqa
+        sys.path.insert(0, lib_path)
```

Comment:
Same comment as above, trust that pyaaf2 has been installed as part of setup.py and we can skip all of this.